### DISCUSSION TITLE
Loosed Domain

### DISCUSSION THREAD

**Discussion by dotsinspace** [vercel/community#3421] [Help]
### Summary

Apologies it's not a technical question.

I was having a domain on bigrock. which I transferred to the vercel. now my domain expired and i can't renew it manually as it has been expired. now i have tried to reach vercel but they have disabled the options of domain support on there website. what to please help.

### Steps to Reproduce

nothing

**Comment by amyegan** [comment]
Hey @dotsinspace. If you can share the domain, then I can look into it for you.

**Comment by taylorjdawson** [comment]
I have a similar issue with an expired domain, however, if you post it publicly you will risk someone else buying it before you correct?

**Reply by dotsinspace** [reply]
Correct. Now i prefer Others like Godaddy etc. instead of Vercel. Horrible experience.
